src/MakeJsonCommand.php

```diff
@@ -80,6 +84,12 @@ public function __invoke( $args, $assoc_args ) {
 			$destination = $args[1];
 		}
 
+		$map = $this->build_map( $map_paths );
+		if ( is_array( $map ) && empty( $map ) ) {
+			WP_CLI::warning( 'No valid keys found. No file was created.' );
+			return;
```

Comment:
If an invalid map causes the script to bail early, this should probably use `WP_CLI::error()` to properly exit the script.